test/functional/wallet_upgradewallet.py

```diff
@@ -73,21 +74,23 @@ def run_test(self):
         res = self.nodes[0].getblockchaininfo()
         assert_equal(res['blocks'], 101)
         node_master = self.nodes[0]
-        v16_3_node  = self.nodes[1]
-        v15_2_node  = self.nodes[2]
+        v16_3_node = self.nodes[1]
+        v15_2_node = self.nodes[2]
 
         # Send coins to old wallets for later conversion checks.
-        v16_3_wallet  = v16_3_node.get_wallet_rpc('wallet.dat')
+        v16_3_wallet = v16_3_node.get_wallet_rpc('wallet.dat')
         v16_3_address = v16_3_wallet.getnewaddress()
         node_master.generatetoaddress(101, v16_3_address)
         self.dumb_sync_blocks()
         v16_3_balance = v16_3_wallet.getbalance()
 
         self.log.info("Test upgradewallet RPC...")
         # Prepare for copying of the older wallet
-        node_master_wallet_dir = os.path.join(node_master.datadir, "regtest/wallets")
-        v16_3_wallet       = os.path.join(v16_3_node.datadir, "regtest/wallets/wallet.dat")
-        v15_2_wallet       = os.path.join(v15_2_node.datadir, "regtest/wallet.dat")
+        node_master_wallet_dir = os.path.join(
+            node_master.datadir, "regtest/wallets")
+        v16_3_wallet = os.path.join(
+            v16_3_node.datadir, "regtest/wallets/wallet.dat")
+        v15_2_wallet = os.path.join(v15_2_node.datadir, "regtest/wallet.dat")
```

Comment:
I don't really like the formatting changes here. This makes it harder to read, imo.

Maybe leave them out for now?

Reply:
All of this formatting was unintentional. I didn't notice it auto formatted the file when I edited the one line above. It will be put back.